src/datagrid/DataGridStoriesData.tsx

```diff
@@ -421,6 +485,29 @@ export const getPageDataForSelectedRows = (pageIndex: number, pageSize: number):
         }, 2000);
     });
 };
+
+// Function to get data for page based on pagenumber
+export const getPageDataForDetailPane = (pageIndex: number, pageSize: number): Promise<DataGridRow[]> => {
+    return new Promise((resolve, reject) => {
```

Comment:
There are some usecases, when we dont have data in Left side pane... in that case the details panel should show some message.  Can we introduce a prop to handle such usecase. Think over it

Usecase:  You  launch Details pane with Namespace which has object stores, once details pane is open, you swich namespace which dont have any object stores.  In that case you done have first row to show data in details pane.  

Worth thinking if fix will be in this component or Parent

Reply:
@maheshkulkarni4 With current implementation of detail pane if you switch namespace when detail pane is open then datagrid will get re-rendered and detail pane will get close.